requirements/default.txt

```diff
@@ -209,6 +209,9 @@ MySQL-python==1.2.3c1
 # sha256: sboSwcRTI_2Glb5oznIoRWuG5RI08WcCfrvEzNqFLGE
 nose==1.1.2
 
+# sha256: REZqm8tW0uVodQ-RUE0SeMdOq7JZowWwbpdbh7UWNdo
```

Comment:
See my comment on the next line. This is here so that we can configure normal test runs to ignore the pytest tests. It should be in `dev.txt`.